apps/workflow-helper.sh

```diff
@@ -71,6 +71,10 @@ docker run --privileged -t --name "tink-worker" \
 	-e "TINKERBELL_CERT_URL=$grpc_cert_url" \
 	-e "REGISTRY_USERNAME=$registry_username" \
 	-e "REGISTRY_PASSWORD=$registry_password" \
+	-e "ROLLBAR_TOKEN=ignored" \
+	-e "ROLLBAR_DISABLE=-1" \
```

Comment:
These were made optional in a recent version of the lib.

Reply:
Sorry, I don't follow. The logger still panics if the token is not set. 
https://github.com/packethost/pkg/blob/master/log/internal/rollbar/rollbar.go#L21

Am I overlooking something here?

Reply:
Opps I thought this was merged already. https://github.com/packethost/pkg/pull/31. Let me check on it and lets hold and wait for it.

Reply:
This looks like an old PR. Is it in shape so we can move it forward?